src/coin/skycoin/api/walletsManager/walletsManager.go

```diff
@@ -0,0 +1,188 @@
+package walletsManager
```

Comment:
Keep this as a flat structure based on API set ... iow In `./src/coin/skycoin/api` there should only be `read.go`, `status.go`, `wallet.go`, `txn.go`, `prometheus.go`, `net_ctrl.go`, `insecure_wallet_seed.go`, `storage.go` them all implementing `api` package.

Reply:
@stdevMauricio1802 this does not even belong in here . No QT classes in plugin folders . This class should work for Skycoin and any other plugin implementing `core.WalletEnv` et al. After reviewing [project folder structure](https://github.com/fibercrypto/FiberCryptoWallet/blob/develop/README.md#project-folder-structure) it seems to me that the right place for this is under `./src/models` .

Reply:
follow a flat structure please , avoid nested hierarchies as much as possible .

Reply:
> Keep this as a flat structure based on API set ... iow In `./src/coin/skycoin/api` there should only be `read.go`, `status.go`, `wallet.go`, `txn.go`, `prometheus.go`, `net_ctrl.go`, `insecure_wallet_seed.go`, `storage.go` them all implementing `api` package.

This is for files under `api` folder